libs/community/langchain_community/llms/__init__.py

```diff
@@ -631,7 +631,7 @@ def __getattr__(name: str) -> Any:
         return _import_aviary()
     elif name == "AzureMLOnlineEndpoint":
         return _import_azureml_endpoint()
-    elif name == "Baichuan":
+    elif name == "BaichuanLLM":
```

Comment:
```suggestion
    elif name == "BaichuanLLM" or name == "Baichuan":
```
no need to make this a breaking change if we don't have to

Reply:
Cool! Great idea. Thanks!